tools/release/breaking-changes.ts

```diff
@@ -0,0 +1,104 @@
+import {join, relative} from 'path';
```

Comment:
I'm not completely sure whether this is proper directory for this script to go in.

Reply:
My opinion is that we should only add it to `tools/release` if it is used by one of the release scripts (staging or publishing). Since this is a standalone script right now, why not move it into `./scripts`?

Reply:
Done.